[ 2024-12-23 19:44:01,222 ] 22 root - INFO - Data Ingestion has been started
[ 2024-12-23 19:44:01,222 ] 24 root - INFO - Data reading using Pandas from local file system
[ 2024-12-23 19:44:01,253 ] 26 root - INFO - Data reading has been completed
[ 2024-12-23 19:44:01,308 ] 30 root - INFO - Raw data has been stored
[ 2024-12-23 19:44:01,314 ] 33 root - INFO - Raw data has been splitted into train and test set
[ 2024-12-23 19:44:01,371 ] 38 root - INFO - Data has been completed
[ 2024-12-23 19:44:01,371 ] 50 root - INFO - Created feature store configuration at /home/azim/mlops-project/feature_repo/feature_store.yaml
[ 2024-12-23 19:44:01,371 ] 54 root - INFO - Configuration file content:
project: income_prediction
provider: local
registry: data/registry.db
online_store:
    type:sqlite
offline_store
    type:file
entity_key_serialization_version: 2
[ 2024-12-23 19:44:01,373 ] 61 root - ERROR - Error in initialization: while scanning a simple key
  in "<unicode string>", line 6, column 1:
    offline_store
    ^
could not find expected ':'
  in "<unicode string>", line 8, column 1:
    entity_key_serialization_version: 2
    ^
[ 2024-12-23 19:44:01,373 ] 52 root - ERROR - 
Error occurred in Python script:
→ File: /home/azim/mlops-project/src/components/data_transformation.py
→ Line number: 57
→ Error message: while scanning a simple key
  in "<unicode string>", line 6, column 1:
    offline_store
    ^
could not find expected ':'
  in "<unicode string>", line 8, column 1:
    entity_key_serialization_version: 2
    ^
